### ISSUE TITLE
Remove or Disable Pinned Your Friend X has just Started Watching Message

### ISSUE THREAD

**Issue by Firedingo** [night/BetterTTV#1786]
I apologize if this is a duplicate but it would be nice to see an option to either hide or otherwise remove the pinned message telling you that a friend has just started watching the same stream as you.
I'm not sure if this is possible either and apologize if I've opened a pointless issue if that is the case.

**Comment by Stoupedog** [comment]
This is already an option, check the bttv settings menu.

**Comment by EhsanKia** [comment]
Is that new? What do you mean pinned message. Can you screenshot it?
As far as I know it's a message in chat, not a pinned message.

**Comment by night** [comment]
If you disable the friends list, we hide the activity. That's the only option we have at the moment.

**Comment by Stoupedog** [comment]
How about the 'Hide Friends Activity in Chat' option?

https://i.imgur.com/3mgfDUl.png

**Comment by Firedingo** [comment]
Apparently somehow my browser caused me to not get an updated version of BTTV so initially when I went looking for that option it didn't show up. After a bit of fiddling around I fixed my browser. Sorry this thread issue appears to be a bit redundant. When I made it I couldn't see an option so I thought it didn't exist. The "Hide Friends Activity in Chat" option seems to do the trick. Apparently my browser thought 6.5 was the most up to date version it could get :/
